virtual_rainforest/models/animals/animal_model.py

```diff
@@ -94,7 +95,9 @@ def from_config(
 
         functional_groups = []
         for k in functional_groups_raw:
-            functional_groups.append(FunctionalGroup(k[0], k[1], k[2], k[3]))
+            functional_groups.append(
+                FunctionalGroup(k[0], k[1], k[2], k[3], k[4], k[5])
```

Comment:
Assuming that checking that the number of columns is the right one has been done somewhere else, then this could be simplified as:
```suggestion
                FunctionalGroup(*k)
```
This will handle possible future situations when the number of arguments changes.

Reply:
It's more robust though to provide these arguments as a dictionary and use `**k`.  That way the order and set of arguments in the signature can change. Don't know how easy it is to change `k` to be a `dict` - and I suspect they're might be performance implications. Thoughts?

Reply:
I doubt it - they are very few arguments to unpack, so overhead should be really minimal.

Keeping that aside, using kwargs would be more robust, that is true.